quarkus-extension/engine/deployment/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+quarkus.arc.auto-inject-fields=false
```

Comment:
I'll move this file into the `/qa/` module in order to provide a nice getting started experience for alpha users. However, we still need to make a decision on how this flag should be configured by default. This will be done with CAM-13748.